Non-CCleaner/Removed entries.ini

```diff
@@ -9,13 +9,62 @@ Detect=HKCU\Software\4Sync
 Default=False
 FileKey1=%AppData%\4Sync|*.log
 
+[Adobe CC Extras *]
+LangSecRef=3023
+DetectFile=%LocalAppData%\Adobe\Creative Cloud Libraries
+Default=False
+FileKey1=%LocalAppData%\Adobe\Creative Cloud Libraries\Logs|*.*
+
+[Adobe CS Extras *]
+LangSecRef=3023
+DetectFile=%AppData%\Adobe\CS*ServiceManager
+Default=False
+FileKey1=%AppData%\Adobe\CS*ServiceManager\Cache|*.*|RECURSE
+FileKey2=%AppData%\Adobe\CS*ServiceManager\Logs|*.*|RECURSE
+FileKey3=%AppData%\com.adobe.downloadassistant.AdobeDownloadAssistant|*.log|RECURSE
+FileKey4=%Documents%\Adobe\Adobe Media Encoder\6.0.0|*.log
+FileKey5=%LocalAppData%\Akamai\Cache|*.*
+FileKey6=%LocalAppData%\Akamai\Logs|*.*|RECURSE
+
 [Adobe Dreamweaver Extras *]
 LangSecRef=3023
 Detect=HKCU\Software\Adobe\MediaBrowser\MRU\Dreamweaver
 Default=False
+FileKey1=%AppData%\Adobe\Dreamweaver CS*\*\Configuration\logs|*.*
 RegKey1=HKCU\Software\Adobe\Dreamweaver CS6\Recent File List
 RegKey2=HKCU\Software\Adobe\MediaBrowser\MRU\Dreamweaver\FileList
 
+[Adobe My Digital Editions Extras *]
+LangSecRef=3023
+DetectFile=%Documents%\My Digital Editions
+Default=False
+FileKey1=%Documents%\My Digital Editions|import.log
+
+[Adobe Photoshop Extras *]
+LangSecRef=3023
+Detect=HKCU\Software\Adobe\Photoshop
+Default=False
+FileKey1=%AppData%\Adobe\Adobe Photoshop*\Generator\logs|*.*
+FileKey2=%AppData%\Adobe\Bridge*\Cache\Thumbnails|*.*|RECURSE
+FileKey3=%AppData%\Adobe\CameraRaw\Cache|*.*|RECURSE
+FileKey4=%AppData%\Adobe\FileBrowser\PhotoshopCS|*.*
+FileKey5=%AppData%\Adobe\Photoshop Album\*.*|Logse*.txt
+FileKey6=%Pictures%|.BridgeSort|RECURSE
+RegKey1=HKCU\Software\Adobe\Photoshop\5.5\VisitedDirs
+RegKey2=HKCU\Software\Adobe\Photoshop\90.0\VisitedDirs
+
+[Adobe Premiere Pro Extras *]
```

Comment:
I think this can just be taken out. This entry refers to pretty old versions of Premiere Pro and there has been many updates since this one that users should have no problem updating to.

Reply:
You are right. FileKey2 probably removes important files and FileKey4+5 are just unnecessary VirtualStore FileKeys.